### ISSUE TITLE
Training accuracy 100%, validation accuracy half of it (near 50%); blank prediction image

### ISSUE THREAD

**Issue by sanjeevan1234** [shekkizh/FCN.tensorflow#64]
I am replicating the same code for binary classification. I added a code to get training accuracy and validation accuracy. But, unfortunately, training accuracy increased to 100% after few iteration but every time, validation accuracy is half of training accuracy.

Can you tell me the possible reason?

Also, I am getting blank prediction image while using inference.

Can you tell me what is the possible reason and solution?

**Comment by dubvulture** [comment]
Did you try lowering the learning rate? (see my older [comment](https://github.com/shekkizh/FCN.tensorflow/issues/54#issuecomment-322314059))

**Comment by lily10086** [comment]
Can you share  me the training accuracy and validation accuracy code??

**Comment by sanjeevan1234** [comment]
Hello, I used this code to get validation and training accuracy.

keep_prob= tf.placeholder(tf.float32, name="keep_probabilty") #Dropout probability
#.........................Placeholders for input image and labels...........................................................................................
    image = tf.placeholder(tf.float32, shape=[None, None, None, 3], name="input_image") #Input image batch first dimension image number second dimension width third dimension height 4 dimension RGB
    GTLabel = tf.placeholder(tf.int32, shape=[None, None, None, 1], name="GTLabel")#Ground truth labels for training

#.........................Build FCN Net...............................................................................................
    Net =  BuildNetVgg16.BUILD_NET_VGG16(vgg16_npy_path=model_path) #Create class for the network
    Net.build(image, NUM_CLASSES,keep_prob)# Create the net and load intial weights
#......................................Get loss functions for neural net work  one loss function for each set of label....................................................................................................
  
#...................................... Get accuracy function for neural network one accuracy function for each set of Label
    correct_prediction = tf.equal(tf.argmax(Net.Prob, 3), tf.argmax(GTLabel, 3))
    accuracy = tf.reduce_mean(tf.cast(correct_prediction, tf.float32))

Thank you.

**Comment by shekkizh** [comment]
Closing due to inactivity
